app/src/main/java/com/checkin/app/checkin/session/activesession/InvoiceOrdersAdapter.kt

```diff
@@ -53,7 +58,15 @@ class InvoiceOrdersAdapter(private var mOrderedItems: List<SessionOrderedItemMod
                     tvName.setCompoundDrawablesWithIntrinsicBounds(if (it) R.drawable.ic_veg else R.drawable.ic_non_veg, 0, 0, 0)
                 else tvName.setCompoundDrawablesWithIntrinsicBounds(0, 0, 0, 0)
             }
-            tvPrice.text = Utils.getCurrencyFormat(itemView.context).format(orderedItem.formatCost())
+            val originalCost: String =Utils.getCurrencyFormat(itemView.context).format(orderedItem.formatOriginalCost())
+            val discountcost: String = Utils.formatCurrencyAmount(itemView.context,orderedItem.formatCost())
+            if(orderedItem.costTampered()){
+                tvPrice.text = StrikethroughSpaninvoice(originalCost, discountcost)
+            }
+           else if(!orderedItem.costTampered()){
+                 tvPrice.text = Utils.getCurrencyFormat(itemView.context).format(orderedItem.formatCost())
```

Comment:
And just set the `actualCost` value here.. No need to format again, since that's already done.